api/src/pcapi/routes/backoffice_v3/filters.py

```diff
@@ -50,14 +52,40 @@ def format_date(data: datetime.date | datetime.datetime, strformat: str = "%d/%m
     return data.strftime(strformat)
 
 
-def format_amount(amount: float) -> str:
+def format_amount(amount: float | None) -> str:
+    if amount is None:
+        amount = 0.0
+
     return f"{amount:,.2f} €".replace(",", "&#8239;").replace(".", ",")
 
 
+def format_bool(data: bool | None) -> str:
+    if data is None:
+        return ""
+
+    if data:
+        return "Oui"
+    return "Non"
+
+
+def parse_referrer(url: str) -> str:
+    """
+    Ensure that a relative is used which, will be understand.
```

Comment:
`Ensure that a relative is used, which will be understood` ? Je ne suis pas 100% certain du message que tu cherches à faire passer, donc pas 100% certain de ma recommandation.